sql/catalyst/src/main/scala/org/apache/spark/sql/AnalysisException.scala

```diff
@@ -48,9 +48,11 @@ class AnalysisException protected[sql] (
 
   // Outputs an exception without the logical plan.
   // For testing only
-  def getSimpleMessage: String = {
+  def getSimpleMessage: String = if (line.isDefined || startPosition.isDefined) {
```

Comment:
this is causing error for where line and startPosition is null. Using spark-sql and spark-core for 3.3.1 and             Spark-catalyst = 3.3.1;